tests/test_helpers.py

```diff
@@ -57,5 +57,46 @@ def test_extract_keyword(self):
             keyword_got = extract_keyword(keyword_line)
 
 
+    def test_find_cross_references(self):
```

Comment:
Can you pull some of these into a parameterized test? The test logic appears to be very similar for multiple.